src/pages/Graph/GraphHelpFind.tsx

```diff
@@ -130,52 +130,52 @@ export default class GraphHelpFind extends React.Component<GraphHelpFindProps> {
     };
   };
 
-  operatorColumns = () => {
+  noteColumns = () => {
     return {
       columns: [
         {
-          property: 'o',
+          property: 't',
           header: {
-            label: 'Operator',
+            label: 'Usage Note',
```

Comment:
note or notes in plural?

Reply:
'Usage Note' is correct here because it is the column header.  The tab is still 'Usage Notes'.